js/src/hooks/useCountryKeyNameMap.js

```diff
@@ -18,10 +30,19 @@ import { useSelect } from '@wordpress/data';
  */
 const useCountryKeyNameMap = () => {
 	return useSelect( ( select ) => {
-		return select( SETTINGS_STORE_NAME ).getSetting(
-			'wc_admin',
-			'countries'
-		);
+		const { getSetting } = select( SETTINGS_STORE_NAME );
+		const countries = {
+			...getSetting( 'wc_admin', 'countries' ),
+		};
+
+		Object.keys( countries ).forEach( ( key ) => {
+			const name = countries[ key ];
+			if ( /&[^;]+;/.test( name ) ) {
```

Comment:
[`@wordpress/html-entities`](https://www.npmjs.com/package/@wordpress/html-entitieshttps://www.npmjs.com/package/@wordpress/html-entities) has a check like this built in: https://github.com/WordPress/gutenberg/blob/f53785fbbf674d46101831206a5154d095e39d0d/packages/html-entities/src/index.js#L19

Reply:
Replaced by https://github.com/woocommerce/google-listings-and-ads/pull/917/commits/2a59a994a56802d1b828c05136cab28683757ced